templates/nextjs/fuels.config.ts

```diff
@@ -0,0 +1,6 @@
+import { createConfig } from 'fuels';
+
+export default createConfig({
+  contracts: ['sway-contracts'],
+  output: './src/sway-api',
+});
```

Comment:
Should we follow the doc's [directory structure](https://fuellabs.github.io/fuels-ts/guide/cli/#getting-started)?
 - `sway-programs`
 - `sway-programs-api`

If not, then the docs should change.

Either way, both must be in accordance.

@FuelLabs/sdk-ts @FuelLabs/frontend Thoughts or preferences?

Reply:
I think the structure mentioned in the docs is the same, it's just the names. 

I should have touched upon this in the PR description - I intentionally chose to use contracts over programs in this quickstart because most devs using this quickstart will be familiar with the term _contracts_ more than _programs_ in the context of a blockchain. It's good if we only introduce them to our contracts first and then somehow slowly introduce our _programs_ - i.e. scripts and predicates.

Reply:
Could the default template receive a flag with what should be included?
 - `--contract` (default)
 - `--predicate`
 - `--script`
 - `--all`

Reply:
Can you please open an issue for this?

Reply:
#1562

Reply:
Thank you.